quiz.js

```diff
@@ -1,57 +1,57 @@
-    to add an overlapping image to make it clear to the user that the car isn not available for 
-    purchase. */
-
-    if(inventory[i].purchased){
-      carString = `<div class="col-sm-4 car sold" style="border: 3px solid ${inventory[i].color};">`;
-      carString += `<h1>${inventory[i].make}</h1>`;
-      carString += `<h2>${inventory[i].model}</h2>`;
-      carString += `<h4>Year: ${inventory[i].year}</h4>`;
-      carString += `<h4>Price: $${inventory[i].price}</h4>`;
-      carString += `<p>${inventory[i].description}</p>`;
-      carString += `<img class="soldImg" src="sold.png">`;
-      carString += `</div>`;
-      htmlContainer.innerHTML += carString;
-
-    } else {
-
-    /* Assuming the purchased key's value is false, this is the format for the div containing the car's
-    information. */
-
-      carString = `<div class="col-sm-4 car" style="border: 3px solid ${inventory[i].color};">`;
-      carString += `<h1>${inventory[i].make}</h1>`;
-      carString += `<h2>${inventory[i].model}</h2>`;
-      carString += `<h4>Year: ${inventory[i].year}</h4>`;
-      carString += `<h4>Price: $${inventory[i].price}</h4>`;
-      carString += `<p>${inventory[i].description}</p>`;
-      carString += `<img class="blankImg" src="blank.png">`;
-      carString += `</div>`;
-      htmlContainer.innerHTML += carString;
-};
+    for (var i = 0; i < inventory.length; i++) {
+
+        /* If the value in the JSON file for the purchased key is true, the div will be built this way
+        to add an overlapping image to make it clear to the user that the car isn not available for 
+        purchase. */
+
+        if (inventory[i].purchased) {
+            carString = `<div class="col-sm-4 car sold" style="border: 3px solid ${inventory[i].color};">`;
+            carString += `<h1>${inventory[i].make}</h1>`;
+            carString += `<h2>${inventory[i].model}</h2>`;
+            carString += `<h4>Year: ${inventory[i].year}</h4>`;
+            carString += `<h4>Price: $${inventory[i].price}</h4>`;
+            carString += `<p>${inventory[i].description}</p>`;
+            carString += `<img class="soldImg" src="sold.png">`;
+            carString += `</div>`;
+            htmlContainer.innerHTML += carString;
+
+        } else {
+
+            /* Assuming the purchased key's value is false, this is the format for the div containing the car's
+            information. */
+
+            carString = `<div class="col-sm-4 car" style="border: 3px solid ${inventory[i].color};">`;
+            carString += `<h1>${inventory[i].make}</h1>`;
+            carString += `<h2>${inventory[i].model}</h2>`;
+            carString += `<h4>Year: ${inventory[i].year}</h4>`;
+            carString += `<h4>Price: $${inventory[i].price}</h4>`;
+            carString += `<p>${inventory[i].description}</p>`;
+            carString += `<img class="blankImg" src="blank.png">`;
+            carString += `</div>`;
+            htmlContainer.innerHTML += carString;
+        };
```

Comment:
This section is not very DRY. If you set a variable to hold either an empty string, "",  or the string "sold", based on whether it's true or not, you can then insert that variable into the "class" on line 16. Then the entire "else" section is unnecessary.